src/Models/UserInput.cs

```diff
@@ -38,7 +39,8 @@ public UserInput
         public string TenantId { get; }
         public KeyValuePair<string, string> Subscription { get; }
         public KeyValuePair<string, string> ResourceGroupName { get; }
-        public KeyValuePair<string, string> AzureMigrateProjectName { get; }
+        public KeyValuePair<string, string> AzureMigrateProjectName { get; }        
+        public bool IsImportDiscovery { get; }
         public string DiscoverySiteName { get; }

```

Comment:
We should not add a special flag for import discovery.

If you look at https://github.com/Azure/azure-migrate-export/blob/main/src/Forms/ConfigurationForm.cs
GetApplianceSources function in the above form .

what we should do is add "import" as any other check box and add functions to disable and uncheck other check boxes if import is selected.

Reply:
In the spec document it was written that the appliance section will get disabled or sth, would modify this section according to the recent mocks.